src/events/ready.ts

```diff
@@ -1,33 +1,42 @@
-import { Client } from 'discord.js';
-import { Config, Interactions } from '../config';
-
+import { ApplicationCommandDataResolvable, Client } from 'discord.js';
 import { Client as Dlist } from 'dlist.js';
+import { Config } from '../config';
 
 export default {
     name: 'ready',
     once: true,
     run: async (client: Client) => {
-        client.application?.commands.set(Interactions);
-        console.log(`\x1b[37mConnected as ${client.user?.tag}\x1b[0m`);
+        const Interactions: ApplicationCommandDataResolvable[] = [];
 
-        const dlist = new Dlist({
-            token: Config.dlist,
-            bot: client.user?.id || '',
+        Config.data.interactions.commands.forEach((command) => {
+            Interactions.push({
+                name: command.name,
+                description: command.description,
+                options: command.options,
+                dm_permission: command.dm_permission,
+            });
         });
 
-        if (process.platform === 'win32') return;
+        client.application?.commands.set(Interactions);
+
+        if (Config.dlist) {
```

Comment:
Why `Config.dlist` if there is the `postStats(client);` few lines beneath it which isn't just dlist. Also why even that if check if it's required inside of the [config.ts](src/example.config.ts), see [typings.ts](src/typings.ts)